security/rbac/README.md

```diff
@@ -133,6 +133,33 @@ confluent iam rolebinding create --principal User:$CLIENT --role ResourceOwner -
 # REST Proxy Admin: no additional administrative rolebindings required because REST Proxy just does impersonation
 
 # Producer/Consumer
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+```
+
+
+## KSQL
+
+* [delta_configs/ksql-server.properties.delta](delta_configs/ksql-server.properties.delta)
+* Role bindings:
+
+```bash
+# KSQL Server Admin
+confluent iam rolebinding create --principal User $ADMIN_KSQL --role ResourceOwner --resource Topic _confluent-ksql-${KSQL_SERVICE_ID}_command_topic --kafka-cluster-id $KAFKA_CLUSTER_ID
```

Comment:
I think the syntax is `Topic:topic-name`, not `Topic topic-name`. If both work, cool :)

Reply:
same for the principal flag

Reply:
Very strange, all the colons got dropped in the copy/paste.  Thanks for noticing this @stan-confluent , I will fix it.